lib/models/movie.js

```diff
@@ -1,14 +1,19 @@
 'use strict';
 
 const Bookshelf = require('../libraries/bookshelf');
+const Location  = require('./location');
 
 module.exports = Bookshelf.Model.extend({
   tableName: 'movies',
+  locations: function () {
+    return this.belongsToMany(Location);
```

Comment:
Did you wind up not needing a `through` argument here?

Reply:
Ended up renaming the table and columns so that they would be the default values bookshelf would look for. It ended up being not necessary.